app/helpers/admin/editions_helper.rb

```diff
@@ -212,4 +212,8 @@ def search_results_table_actions(edition)
 
     sanitize(actions)
   end
+
+  def edition_title_or_edition_title_link(edition)
```

Comment:
It might be a bit clearer if the logic is reversed here, either in the name of the method or the implementation i.e. `edition_title` is sort of the second return value, but the first part of the method name if that makes sense?

Reply:
Yeah, I did have this thought but I was falling over all the words! `edition_title_link_or_edition_title` is probably okay though?